c/examples/BUILD

```diff
@@ -0,0 +1,19 @@
+package(default_visibility = ["//visibility:public"])
+
+# Simple client example with no connection error detection logic.
+cc_binary(
+    name = "simple_polaris_client",
+    srcs = ["simple_polaris_client.c"],
+    deps = [
+        "//:polaris_client",
+    ],
+)
+
+# Example application including connection timeout and retry logic.
+cc_binary(
+    name = "connection_retry",
```

Comment:
I think it'd be nice if the examples all have a suffix of "client" for consistency

Reply:
Sure